### PR TITLE
Remove StyleCop references

### PR DESCRIPTION
**Pull request by jaredpar**
The code base no longer uses StyleCop.  Removing lingering references to
it from the code base.

### CODE REVIEW

File: src/Compilers/Core/AnalyzerDriver/AnalyzerDriver.projitems [hunk 1]
```diff
@@ -9,23 +9,11 @@
     <Import_RootNamespace>AnalyzerDriver</Import_RootNamespace>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="$(MSBuildThisFileDirectory)AnalyzerDriverHelper.cs">
```
Comment (srivatsn) [inline src/Compilers/Core/AnalyzerDriver/AnalyzerDriver.projitems]: I think there are a lot ExcludeFromStyleCop directives in a bunch of other project files. Can you clean those up too?

Comment (jaredpar) [inline src/Compilers/Core/AnalyzerDriver/AnalyzerDriver.projitems]: I did an exhaustive search for StyleCop in the GitHub code and it came up clean after this change.  Going to do a corresponding one in TFS once this is submitted.

Comment (srivatsn) [inline src/Compilers/Core/AnalyzerDriver/AnalyzerDriver.projitems]: Ah, looks like you removed them from csprojs when you did the pass to apply the style conventions. Workspaces.csproj some for example has some that were removed.

**Comment by srivatsn** [comment]
:+1:

File: src/Compilers/Core/AnalyzerDriver/AnalyzerDriver.projitems (hunk 1 above)
Comment (jaredpar) [inline src/Compilers/Core/AnalyzerDriver/AnalyzerDriver.projitems]: Yeah I got some during the style conversion but missed a few others.

**Comment by tmat** [comment]
Finally :)

**Pull request merged**